source/includes/usage-examples/read-write-pref.rb

```diff
@@ -0,0 +1,80 @@
+require 'bundler/inline'
+
+gemfile do
+  source 'https://rubygems.org'
+  gem 'mongo'
+end
+
+uri = '<connection string>'
+
+Mongo::Client.new(uri) do |client|
+    # Access the database and collection
+
+    # start-write-concern
+    client = Mongo::Client.new(['IP_ADDRESS_001:27017'], database: 'myDB')
+    myDB = client.database
+    myCollection = myDB[:myCollection]
+
+    myCollection.insert_one(
+      { name: "anotherDocumentName" },
```

Comment:
You're using single-quoted strings above this; it's probably best to be consistent and stick to single-quoted strings everywhere, unless double-quoted strings are called for (e.g. for string interpolation).